src/test/java/com/kobylynskyi/graphql/codegen/TestUtils.java

```diff
@@ -23,4 +26,17 @@ public static void assertSameTrimmedContent(File expected, File file) throws IOE
         String actualContent = Utils.getFileContent(file.getPath()).trim();
         assertEquals(expectedContent, actualContent);
     }
+
+    public static MappingConfig initMappingConfig() {
+        MappingConfig mappingConfig = new MappingConfig();
+        mappingConfig.setSerialVersionUIDGenerator(getSerialVersionUIDGenerator());
+        return mappingConfig;
+    }
+
+    public static SerialVersionUIDGenerator getSerialVersionUIDGenerator() {
+        SerialVersionUIDGenerator serialVersionUIDGenerator = Mockito.mock(SerialVersionUIDGenerator.class);
+        Mockito.when(serialVersionUIDGenerator.randomLong()).thenReturn(123456L);
```

Comment:
This mock actually shows the problem. Serial version UID should be deterministic and not require mocking

Reply:
The idea of hardcoding serialVersionUID here was because we need to compare generated class with the expected one.
But this mocking can be removed since explicit serialVersionUID is not recommended, right?